### DISCUSSION TITLE
When can i ge student benefits??

### DISCUSSION THREAD

**Discussion by appoung** [community/community#135588] [GitHub Education]
### Hi everyone,

![IMG_4929](https://github.com/user-attachments/assets/06e0944b-724a-41f2-8131-be43beb981c1)

**Comment by davevad93** [comment]
Hi @appoung ,

Waiting times are approximate and may vary depending on different factors, such as the volume of requests. It usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @appoung was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135145

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
